hikari/impl/event_manager_base.py

```diff
@@ -71,51 +71,23 @@ class EventManagerBase(event_manager.EventManager):
     is the raw event name being dispatched in lower-case.
     """
 
-    __slots__: typing.Sequence[str] = ("_app", "_consumers", "_intents", "_listeners", "_waiters")
+    __slots__: typing.Sequence[str] = ("_app", "_listeners", "_consumers", "_waiters")
 
     def __init__(self, app: traits.BotAware) -> None:
         self._app = app
-        self._consumers: typing.Dict[str, typing.List[ConsumerT]] = {}
-        self._intents = app.intents
+        self._consumers: typing.Dict[str, ConsumerT] = {}
         self._listeners: ListenerMapT[base_events.Event] = {}
         self._waiters: WaiterMapT[base_events.Event] = {}
 
         for name, member in inspect.getmembers(self):
             if name.startswith("on_"):
-                self.add_raw_consumer(name[3:], member)
-
-    def add_raw_consumer(self, name: str, consumer: ConsumerT, /) -> None:
-        name = name.casefold()
-        if name not in self._consumers:
-            self._consumers[name] = []
-
-        self._consumers[name].append(consumer)
-
-    def get_raw_consumers(self, name: str, /) -> typing.Sequence[ConsumerT]:
-        if consumers := self._consumers.get(name.casefold()):
-            return consumers.copy()
-
-        return []
-
-    def remove_raw_consumer(self, name: str, consumer: ConsumerT, /) -> None:
-        name = name.casefold()
-
-        try:
-            self._consumers[name].remove(consumer)
-
-        except ValueError:
-            raise LookupError(name, consumer) from None
-
-        if not self._consumers[name]:
-            del self._consumers[name]
+                self._consumers[name[3:]] = member
```

Comment:
oh yeah, you did make sure that this behaviour is working right with that casefold down in consume_raw_event right?

Reply:
Yep :+1: